### DISCUSSION TITLE
how to tp mobs/nutcracker outside?

### DISCUSSION THREAD

**Discussion by joep26020** [winstxnhdw/lc-hax#179] [Q&A]
### What happened?

4r5 1555

### Current Commit Hash

5555523

### Injector

- [ ] SharpMonoInjectorCore
- [ ] Others

### Acknowledgement

- [X] I have **recursively** cloned the repository as instructed in the [README](https://github.com/winstxnhdw/lc-hax/edit/main/README.md)
- [X] I have done my due diligence to check for similar issues

**Comment by winstxnhdw** [comment]
`/mob <player>`
